### PR TITLE
Controller specs

### PR DESCRIPTION
**Pull request by frempongdev**
## In this Milestone, I made the following additions ➕ ➕ 🟢
- ###  Set up tests on `User` and `Post` controllers.
  - [Added request spec for Users controller](https://github.com/frempongdev/blog-app/commit/e25277863fa899e214b53da26b7c23e3159e09c2)
  - [Added request spec for Posts controller](https://github.com/frempongdev/blog-app/commit/1ae9abd6913fa6b480572e8b77482b5ce73aee94)
  - [Fixed Linter errors](https://github.com/frempongdev/blog-app/commit/797981b7fe03a2e28522b009a707dd02ea1ccf79)🧰

### CODE REVIEW

File: spec/requests/users_spec.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  it ' returns http status ok for /user/:id' do
+    get users_path user.id
+    expect(response.status).to eq(200)
+    expect(response.status).not_to eq(404)
+  end
+
+  it 'renders the show template for users controller' do
+    get user_path(user.id)
+    expect(response).to render_template(:show)
+  end
```
Comment (thecodechaser) [inline spec/requests/users_spec.rb:33]: - [x] Please make sure to add the below test for users#show action as requested in the project requirements.
image

File: spec/requests/posts_spec.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  it ' returns http status ok for /user/:id/posts/:id' do
+    get user_posts_path(user.id)
+    expect(response.status).to eq(200)
+    expect(response.status).not_to eq(404)
+  end
+
+  it 'renders the show template' do
+    get user_post_path(user.id, post.id)
+    expect(response).to render_template(:show)
+  end
```
Comment (thecodechaser) [inline spec/requests/posts_spec.rb:37]: - [x] Please make sure to add the below test for posts#show action as requested in the project requirements.
image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @frempongdev👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- All tests are passed. ✔️
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/requests/posts_spec.rb (hunk 2 above)
Comment (frempongdev) [reply spec/requests/posts_spec.rb:37]: Hey @thecodechaser, 
Thank You for turning my head toward that section of the project requirement.
This is a great tip that will help me in the long run. 👌🏿

File: spec/requests/users_spec.rb (hunk 1 above)
Comment (frempongdev) [reply spec/requests/users_spec.rb:33]: I will certainly do this. 😃

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @frempongdev👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- All tests are passed. ✔️
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by frempongdev** [comment]
I appreciate your efforts, @thecodechaser 🙏🏿

**Pull request merged**